.yetus/detsecrets-ignored-hashes.txt

```diff
@@ -26,3 +26,4 @@ d8e18255635efa84e79bb9eaf2ffad3371a95288
 3b3c84f10a63cb8e95b93ea5f46e02d885153f5d
 c267b646441a206d44803d8cb20896c4a166cac2
 f4d95a7d14e9c4c69afe28d54743222acbe04b74
+66defd532d6875b42111a786d7bf52bb8db61602
```

Comment:
What is this file? Do we ever remove entries from it?

Reply:
> What is this file?

It is the exemption file for [detect-secrets](https://yetus.apache.org/documentation/0.14.0/precommit/plugins/detsecrets/). For Yetus, this mostly contains shas of other shas that detect-secrets finds.  I think there are some references to password variables in precommit as well.

> Do we ever remove entries from it?

We haven't yet, but we could.  It was just introduced as a new feature in 0.14 so shiny new.